bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -264,6 +299,12 @@ All these channels are read only.
 
 All these channels are read only.
 
+**Extensible channels for the rain guage:**
+
+| Channel Type         | Item Type            | Description                       | Channel parameters                                 |
+|----------------------|----------------------|-----------------------------------|----------------------------------------------------|
+| sum_rain-measurement | Number:Length        | Summing rain measurement          | period (30min, 1hour, 3hours, 1day, 1week, 1month) |
```

Comment:
Is it really **sum_rain** with underscore? Just checking, as it seems inconsistent with convention used.

Reply:
Yes.
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/NetatmoConstants.java#L87

Reply:
Now wondering if this is by mistake, and if we should change it to "sum-rain" for consistency? In 3.3 all channels are renamed anyway, so last chance to fix this without further breaking. @clinique - is it by accident, or by purpose that underscore is used here?

Reply:
Yes, it is not very nice but by design because I use Netatmo API keyword